src/chip.css.ts

```diff
@@ -66,6 +66,29 @@ export const chipVariants = styleVariants({
             cursor: 'pointer',
         },
     ],
+    navigationActive: [
+        chipActive,
+        containerBase,
+        sprinkles({
+            color: vars.colors.textPrimaryInverse,
+            background: vars.colors.controlActivated,
+        }),
+        {
+            borderColor: vars.colors.controlActivated,
+            cursor: 'pointer',
+        },
+    ],
+    navigationActiveInverse: [
+        containerBase,
+        sprinkles({
+            color: vars.colors.controlActivated,
```

Comment:
```suggestion
            color: vars.colors.textActivated,
```

I found a mistake in specs, please, fix this color

Reply:
done